cmd/bootnode.go

```diff
@@ -133,7 +135,19 @@ func RunBootnode(ctx context.Context, config BootnodeConfig) error {
 			}
 		}
 
-		tcpNode, err := p2p.NewTCPNode(config.P2PConfig, key, p2p.NewOpenGater(), udpNode, nil, nil)
+		// Increase resource limits
+		const maxConns = 1024
```

Comment:
can we add a flag for this in bootnode command?

Reply:
+1 for flag on the bootnode command. Its more 12factor than code commits for config changes. Maybe something like `--max-peers`?

Reply:
I have added it as `--p2p-max-connections` with description: "Libp2p maximum number of peers that can connect to this bootnode."